scripts.py

```diff
@@ -232,6 +235,10 @@ def arg_parser(cls):
             help='Process identifiers of this type. If IDENTIFIER is not specified, all identifiers of this type will be processed. If IDENTIFIER is specified, this argument is required.'
         )
         parser.add_argument(
+            '--identifier-data-source',
+            help='Process identifiers with this LicensePool.data_source'
```

Comment:
This is a little confusing to me. I would say "Process only identifiers which have a license pool associated with this data source."